coreapps-web-2.2/src/test/java/org/openmrs/module/coreapps/diagnosisCompatibility/DiagnosisServiceCompatibility2_2Test.java

```diff
@@ -0,0 +1,37 @@
+package org.openmrs.module.coreapps.diagnosisCompatibility;
+
+
+
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.Diagnosis;
+import org.openmrs.Encounter;
+import org.openmrs.api.EncounterService;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+public class DiagnosisServiceCompatibility2_2Test  extends BaseModuleContextSensitiveTest {
+	
+	@Autowired
+	@Qualifier("encounterService")
+	EncounterService encounterService;
+	
+	@Autowired
+	DiagnosisServiceCompatibility diagnosisServiceCompatibility;
+	
+	/**
+	 * @throws Exception 
+	 * @see DiagnosisServiceCompatibility2_2#getDiagnoses(Encounter)
+	 */
+	@Test
+	public void getDiagnoses_shouldGetDiagnosesOfTheProvidedEncounter() throws Exception {
+		Encounter encounter = encounterService.getEncounter(3);
+		List<Diagnosis> list = diagnosisServiceCompatibility.getDiagnoses(encounter);
+		Assert.assertNotNull(list.size());
```

Comment:
There's so many ways that the size might not be null **and** the diagnoses to not be those expected.

In other words: assert that you actually get the diagnoses that are expected.